pennylane/transforms/sign_expand_data.json

```diff
@@ -0,0 +1,902 @@
+[
```

Comment:
I believe it will be tidier to create a `sign_expand` folder inside the `transforms` folder and move `sign_expand.py` and `sign_expand_data.json` inside it.

@albi3ro do you agree?

Reply:
Moved into its own folder